src/components/AppController/AppController.js

```diff
@@ -14,12 +14,7 @@ var App = React.createClass({
 
   getInitialState: function() {
     return {
-      meta: [],
-      filter: {},
-      model_id: 'cgcm3',
-      variable_id: 'tasmax',
-      experiment: 'rcp45',
-      area: undefined
+      meta: []
```

Comment:
So, I know it's a bit verbose and functionally unnecessary to declare our state variables here as empty or undefined. But OTOH, I think we need *something* at the top of the file for our devs to know what they can expect the component's state to be. Could be a docstring, but failing that, I propose we leave the empty declarations.